pmaweb/views.py

```diff
@@ -50,6 +50,7 @@
     'about': 'about',
     'awards': 'awards',
     'contest': 'contest',
+    'contractor': 'contractor',
```

Comment:
This is really just a compatibility mapping of old website URLs, no need to add anything here. I've improved the docs in 5294675.